pkg/genericapiserver/options/etcd.go

```diff
@@ -83,4 +83,7 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 
 	fs.BoolVar(&s.StorageConfig.Quorum, "etcd-quorum-read", s.StorageConfig.Quorum,
 		"If true, enable quorum read.")
+
+	fs.BoolVar(&s.StorageConfig.EnableCURLDebug, "enable-curl-debug", s.StorageConfig.EnableCURLDebug,
```

Comment:
Sorry I didn't notice that, I was curious why there was an unused variable. Yes it's better not to expose this option. 
I will close this pr, thanks for your reply!